tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py

```diff
@@ -95,32 +97,11 @@ class MarketplaceDev(Marketplace):
     name = 'Dev'
     manifest_url = 'https://marketplace.firefox.com/app/bee10fbb-7829-4d99-8a9f-a6d0a905138e/manifest.webapp'
 
-    _enable_addons_locator = (By.ID, 'enable-addons')
-
-    def _enable_addons(self):
-        enable_addon_button = Wait(self.marionette).until(
-            expected.element_present(*self._enable_addons_locator))
-        Wait(self.marionette).until(expected.element_displayed(enable_addon_button))
-        enable_addon_button.tap()
-
-    def _open_addon(self, addon_name):
-        selector = '//a[descendant::*[text()="%s"]]' % addon_name
-        addon_locator = (By.XPATH, selector)
-
-        addon_link = Wait(self.marionette, timeout=20, interval=1).until(
-            expected.element_present(*addon_locator))
-        Wait(self.marionette).until(expected.element_displayed(addon_link))
-        self.marionette.execute_script('arguments[0].scrollIntoView(true);', [addon_link])
-        addon_link.tap()
-        return AddonPage(self.marionette)
-
     def install_addon(self, addon_name):
-        # Open debug page
-        self.search(':debug')
-
-        self._enable_addons()
+        # Shortcut, go directly to the Addon page
+        self.marionette.navigate('https://marketplace-dev.allizom.org/addon/%s' % addon_name)
```

Comment:
Indeed, Marketplace-Dev changes a lot, this would safe me from fixing this test once every week or so.
And we only want to test the ability of installing an add-on susccesfully from Marketplace-Dev anyway.
The UI aspects of Marketplace-Dev is handled by the Marketplace team.